src/elements/StatusBadge/StatusBadge.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Status Badge
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import Common from '../Common/Common';
+
+const StatusBadge = ({ children, active, paused, pending, archived, ...other }) => {
+
+	const combinedClasses = classNames(
+		'status-badge',
+		active && 'active',
```

Comment:
There's a possibility that someone could accidentally pass in multiple of these props, which would add multiple class names as well as display multiple lines of text (lines 29-32). Instead it might be better to have one prop called `status`, which will accept one of those values (`active`, `paused`, etc). You can then use that `status` prop to both define the class as well as set the text in the `return` part of the component. So you'd add `status && status` in the `combinedClasses` variable instead of the individual conditions, and do something like `{status && status.toUpperCase()}` in the return for the text.

You could then reinforce this idea that this prop needs to have one of a set number of string values with a custom prop type like this:

```js
StatusBadge.propTypes = {
	status: (props, propName, componentName) => {
		if(!['active', 'paused', 'pending', 'archived'].includes(props.status)) {
			return new Error(`${componentName} requires "${propName}" prop to be either "active", "paused", "pending", or "archived"`);
		}
	}
};
```

Reply:
I was following the pattern used in the button, which does the same thing:

```		small && 'small',
		mini && 'mini',
		full && 'full',
		loading && 'loading'
```

but I'll go ahead and change it.